dds/DCPS/Service_Participant.h

```diff
@@ -434,6 +434,11 @@ class OpenDDS_Dcps_Export Service_Participant {
   DDS::Duration_t bit_autopurge_disposed_samples_delay() const;
   void bit_autopurge_disposed_samples_delay(const DDS::Duration_t& duration);
 
+  enum TypeObjectEncoding { Encoding_Normal, Encoding_WriteOldFormat, Encoding_ReadOldFormat };
```

Comment:
`Encoding_ReadOldFormat` seems not used.

Reply:
It is indirectly, since there is a state that is ` != Service_Participant::Encoding_Normal` and also `!= Service_Participant::Encoding_WriteOldFormat`